src/coreclr/classlibnative/inc/floatdouble.h

```diff
@@ -9,6 +9,11 @@
 
 class COMDouble {
 public:
+    struct SinCosResult {
+        double Sin;
+        double Cos;
+    };
```

Comment:
Worth a comment that this needs to have the same layout as `ValueTuple<double, double>`?

Reply:
Added.